index/generator/tests/registry/samples/code-with-quarkus/devfile.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
   tags: ["Java", "Quarkus"]
   projectType: "quarkus"
   language: "java"
+  provider: "Red Hat"
```

Comment:
`schemaVersion` for these test devfiles are 2.0.0, shouldn't they be `2.2.0-latest`?

Reply:
parser do not support format of `<version>-<tag>`. so updated to `2.2.0` instead